site/en/docs/extensions/mv3/tut_websockets/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Using WebSockets in Service Workers'
+seoTitle: 'How to use WebSockets in Chrome Extension Service Workers'
+date: 2023-07-04
+description: 'Step-by-step instructions on how to connect to a WebSocket in your Chrome Extension.'
+---
+
+This tutorial demonstrates how to connect to a WebSocket in your Chrome Extension's service worker. You can find a [working example on Github](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/tutorial.websockets).
+
+## Background
+
+Starting with Chrome 116, extension service workers get improved support for WebSockets. Previously, a service worker could become inactive despite a WebSocket connection being active if no other extension events occurred for 30 seconds, which would result in the service worker being terminated and the WebSocket connection being closed.
```

Comment:
```suggestion
Starting with Chrome 116, extension service workers get improved support for WebSockets. Previously, a service worker could become inactive despite a WebSocket connection being active if no other extension events occurred for 30 seconds. This would terminate the service worker and close the WebSocket connection.
```
Readability tweak